audit_exceptions/permitted_formula_license_mismatches.json

```diff
@@ -1 +1,3 @@
-[]
+[
+  "pegtl"
```

Comment:
Does our audit not work? We run the check on a specific tag so it should automatically fail when tag uses a new license
```console
❯ gh api '/repos/taocpp/PEGTL/license?ref=3.2.7' | jq .license.spdx_id
"MIT"

❯ gh api '/repos/taocpp/PEGTL/license?ref=c7630f1' | jq .license.spdx_id
"BSL-1.0"
```